misc/windowswitcher.talon

```diff
@@ -0,0 +1,2 @@
+switch window:  user.show_window_switcher()
+switch <user.window_selection_word>: user.switch_to_window(window_selection_word)
```

Comment:
How would you feel about using <numbers> here instead?

With an upcoming PR, you'd be able to say either the whole number, or the digits, to switch windows. Similar to file manager.